.github/pull_request_template.md

```diff
@@ -1,29 +1,30 @@
 ## Proposed changes
 
-Describe the big picture of your changes here to communicate to the maintainers why the pull request is necessary and why we should accept it. If it fixes a previously reported bug, be sure to link to that issue. Please also summarize the changes to the code.
+Contributions to PorePy are highly appreciated. Clearly explain to the maintainers why this pull request (PR) is needed and why it should be accepted. If this PR solves an issue, explain how it is done. Please, also summarise the changes to the code. 
 
 ## Types of changes
 
-What types of changes does your code introduce to PorePy?
+What types of changes does this PR introduce to PorePy?
 _Put an `x` in the boxes that apply_
 
+- [ ] Minor change (e.g., dependency bumps, broken links, etc.)
```

Comment:
This list has no punctuation marks, but the list below does.

Reply:
Punctuation marks were included.